CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- [Issue #372](https://github.com/manheim/terraform-pipeline/issues/372) Replace `master` branch with `main` in code
```

Comment:
This isn't accurate right?  `master` was replaced with `main` in the docs, but both are supported functionally, correct? 

Can the Issue and the CHANGELOG be updated to reflect the change?  eg: "ConditionalApplyPlugin: Support `main` and `master` branches."

Reply:
AH! Yes, I can do that.